front/lib/auth.ts

```diff
@@ -429,77 +437,75 @@ export async function getAPIKey(
   return new Ok(key);
 }
 
-const DEFAULT_DATASOURCES_COUNT_LIMIT = 1;
-const DEFAULT_DATASOURCES_DOCUMENTS_COUNT_LIMIT = 32;
-const DEFAULT_DATASOURCES_DOCUMENTS_SIZE_MB_LIMIT = 1;
-
 /**
  * Construct the PlanType for the provided workspace.
  * @param w WorkspaceType the workspace to get the plan for
  * @returns PlanType
  */
-export function planForWorkspace(w: Workspace): PlanType {
-  const limits = {
-    dataSources: {
-      count: DEFAULT_DATASOURCES_COUNT_LIMIT,
-      documents: {
-        count: DEFAULT_DATASOURCES_DOCUMENTS_COUNT_LIMIT,
-        sizeMb: DEFAULT_DATASOURCES_DOCUMENTS_SIZE_MB_LIMIT,
-      },
-      managed: false,
-    },
-    largeModels: false,
-  };
+export async function planForWorkspace(
+  w: Workspace
+): Promise<Promise<PlanType>> {
+  let activeSubscription: Subscription | null = null;
+  const today = new Date();
```

Comment:
I did not want to have a nullable start_date to simplify the code. 
Using `createdAt` is not the truth in case the workspace was upgraded than downgraded.
Using `today` is also not the truth, I thought it was less bad than `createdAt` as easier to understand it's a default value. 

If we want it clean we would have to compute: "day after last ended subscription or workspace.createdAt". This looks quite costly for an information we don't use in the app, so I will make it nullable.

Reply:
I added a commit for this 🙏🏻